src/components/Accordion.js

```diff
@@ -32,21 +72,27 @@ const selectElementByClassName = (element, cssName) => {
   }
 };
 
-const menuAction = element => {
-  var mainDiv = element.closest(".dg_accordion");
+const menuAction = accordionHeaderElm => {
+  var mainDiv = accordionHeaderElm.closest(".dg_accordion");
   var menuItems = mainDiv.getElementsByClassName("multi-collapse");
   var totalCollapsed = mainDiv.getElementsByClassName("show");
-  var buttonElement = element.closest("button");
-  var childDiv = selectElementByClassName(buttonElement, ".multi-collapse");
-  var buttonAll = mainDiv.getElementsByClassName("dg_allitems");
-
-  var isMenuOpen = childDiv.className.includes(menuOpen);
+  var accordionButton = accordionHeaderElm.closest("button");
```

Comment:
Not blocking @tmccoy529, but you killed yourself on this one, but it seems like you were ok with the pattern suggestion. But you only applied it to the param which is great. But you missed the suffixes on other variables.

Reply:
@tmccoy529 updated this so I'm resolving.